_layouts/candidate.html

```diff
@@ -46,26 +46,29 @@ <h2>{{ candidate.name }}</h2>
         <h2 class="candidate__expenditures-opposing">Expenditures opposing candidate <span class="money">{{ candidate.opposing_money.opposing_expenditures | times: 100 | money }}</span></h2>
       </div>
     </section>
+    {% assign money_bar_max = candidate.total_contributions | plus: candidate.total_expenditures %}
```

Comment:
Should this be the max of the items that represent the money bar? i.e. the categories themselves.

Reply:
The recommendation from the [usability report](https://docs.google.com/document/d/1YRlgHC-Venn5oaTY7jrk42OJfabM8171EykkcHC1EQE/edit#heading=h.ph9yvg64cyeb) was that we should make the bars comparable between contributions and expenditures if put side by side. So if I'm understanding correctly, that means we should be using the same maximum between the categories and summing them seems like the best way to do that.

Instead of using the sum, we could use the maximum of the individual entries. That would make the entry with the highest value appear 100% and all the others would be lower. This would reduce the white space you see in the bars. (and as I'm explaining this, its starting to sound more like the correct solution :thinking: )

Reply:
> Instead of using the sum, we could use the maximum of the individual entries.

Going to merge this as is for now. I _think_ this can be done in the Liquid template language, but it's a bit tricky to pull the values out of the object. I think I need to loop over both the objects' properties myself to calculate the max.